src/components/structures/MatrixChat.js

```diff
@@ -847,16 +847,37 @@ export default React.createClass({
         }).close;
     },
 
+    _leaveRoomWarnings: function(roomId) {
+        const roomToLeave = MatrixClientPeg.get().getRoom(roomId);
+        // Show a warning if there are additional complications.
+        const roomState = roomToLeave.getLiveTimeline().getState("f");
```

Comment:
ooi why this vs `roomToLeave.currentState` ?

Reply:
oh RIP
> The state of the room at the time of the newest event in the timeline. Present for backwards compatibility - prefer getLiveTimeline().getState(false).

I've been using currentState for ages :L
--

Reply:
turns out that doc is wrong because false !== 'f'

Reply:
![screenshot from 2018-01-19 22-46-31](https://user-images.githubusercontent.com/2072976/35175239-a21cff86-fd6a-11e7-8594-bc1557b4d8e9.png)

I'm guessing the docs are lying?

Reply:
Yah thats why I said
> turns out that doc is wrong because false !== 'f'

 
I'll look into fixing that

Reply:
GH doesn't update comments quick enough. Anyway will fix indentation + using currentState instead.

Reply:
Noooo, docs say prefer getState, the bit thats wrong is the arg
I'm lazy and use currentState, but if docs say prefer yours then you do you